platform/viewer/src/lib/DICOMSR/dataExchange.js

```diff
@@ -0,0 +1,42 @@
+import OHIF from '@ohif/core';
```

Comment:
Everything related to DICOMSR is pretty critical. It would be nice to see unit tests either before merge, or as an immediate next task.